mod/invite.php

```diff
@@ -119,7 +119,7 @@ function invite_content(&$a) {
 		if($a->config['register_policy'] == REGISTER_CLOSED)
 			$linktxt = sprintf( t('Visit %s for a list of public sites that you can join. Friendica members on other sites can all connect with each other, as well as with members of many other social networks.'), $dirloc . '/siteinfo');
 		elseif($a->config['register_policy'] != REGISTER_CLOSED)
```

Comment:
Standards: Could you please review this whole conditional statement for brackets and spaces? 😊